libs/coin-modules/coin-aptos/src/api/index.ts

```diff
@@ -30,6 +30,7 @@ import {
   GetAccountTransactionsDataQuery,
   GetAccountTransactionsDataGtQueryVariables,
 } from "./graphql/types";
+import { TokenCurrency } from "@ledgerhq/types-cryptoassets";
```

Comment:
please see my comment [here](https://github.com/LedgerHQ/ledger-live/pull/9424/files#r1987765948) as it turns out you need as a `dep` and not `devDep`